### PR TITLE
Spanish translation of the file multilingual.mdx

### PR DESCRIPTION
**Pull request by SimplyJuanjo**
# What does this PR do?

Adds the Spanish version of [multilingual.mdx](https://github.com/huggingface/transformers/blob/master/docs/source/en/multilingual.mdx) to [transformers/docs/source_es](https://github.com/huggingface/transformers/tree/main/docs/source/es)

Fixes https://github.com/huggingface/transformers/issues/15947 (issue)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?

## Who can review?

@omarespejel @osanseviero

### CODE REVIEW

File: docs/source_es/multilingual.mdx [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+```py
+>>> import torch
+>>> from transformers import XLMTokenizer, XLMWithLMHeadModel
+
+>>> tokenizer = XLMTokenizer.from_pretrained("xlm-clm-enfr-1024")
+>>> model = XLMWithLMHeadModel.from_pretrained("xlm-clm-enfr-1024")
+```
+
+El atributo `lang2id` del tokenizador muestra los idiomas de este modelo y sus identificadors:
```
Comment (omarespejel) [inline docs/source_es/multilingual.mdx:47]:
```suggestion
El atributo `lang2id` del tokenizador muestra los idiomas de este modelo y sus identificadores:
```

File: docs/source_es/multilingual.mdx [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+Los siguientes modelos de MBart pueden utilizarse para traducción multilingüe:
+
+- `facebook/mbart-large-50-one-to-many-mmt` (One-to-many multilingual machine translation, 50 languages)
+- `facebook/mbart-large-50-many-to-many-mmt` (Many-to-many multilingual machine translation, 50 languages)
+- `facebook/mbart-large-50-many-to-one-mmt` (Many-to-one multilingual machine translation, 50 languages)
+- `facebook/mbart-large-50` (Multilingual translation, 50 languages)
+- `facebook/mbart-large-cc25`
+
+En este ejemplo, carga el checkpoint `facebook/mbart-large-50-many-to-many-mmt` para traducir el finlandés al inglés. Puedes establecer el idioma de origen en el tokenizador:
```
Comment (omarespejel) [inline docs/source_es/multilingual.mdx:149]:
```suggestion
En este ejemplo, carga el checkpoint `facebook/mbart-large-50-many-to-many-mmt` para traducir del finlandés al inglés. Puedes establecer el idioma de origen en el tokenizador:
```

**Review by omarespejel — CHANGES_REQUESTED** [review]
Thank you! Some nits

File: docs/source_es/multilingual.mdx [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+>>> model = AutoModelForSeq2SeqLM.from_pretrained("facebook/mbart-large-50-many-to-many-mmt")
+```
+
+Tokeniza el texto:
+
+```py
+>>> encoded_en = tokenizer(en_text, return_tensors="pt")
+```
+
+MBart fuerza el id del idioma de destino como el primer token generado para traducir al idioma de destino. Establezca el `forced_bos_token_id` como `en` en el método `generate` para traducir al inglés:
```
Comment (omarespejel) [inline docs/source_es/multilingual.mdx:167]:
```suggestion
MBart fuerza el id del idioma de destino como el primer token generado para traducirlo. Establece el `forced_bos_token_id` a `en` en el método `generate` para traducir al inglés:
```

File: docs/source_es/multilingual.mdx [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+>>> model = M2M100ForConditionalGeneration.from_pretrained("facebook/m2m100_418M")
+```
+
+Tokeniza el texto:
+
+```py
+>>> encoded_zh = tokenizer(chinese_text, return_tensors="pt")
+```
+
+M2M100 fuerza el id del idioma de destino como el primer token generado para traducir al idioma de destino. Establezca el `forced_bos_token_id` como `en` en el método `generate` para traducir al inglés:
```
Comment (omarespejel) [inline docs/source_es/multilingual.mdx:131]:
```suggestion
M2M100 fuerza el id del idioma de destino como el primer token generado para traducirlo. Establece el `forced_bos_token_id` a `en` en el método `generate` para traducir al inglés:
```

**Review by omarespejel — CHANGES_REQUESTED** [review]

File: docs/source_es/multilingual.mdx (hunk 2 above)
Comment (SimplyJuanjo) [reply docs/source_es/multilingual.mdx:149]: In this case,  maybe the source/eng should be corrected too, a "from" before "Finnish" is missing in the original english docs.

**Comment by SimplyJuanjo** [comment]
any other thing left to do? @omarespejel

**Review by omarespejel — APPROVED** [review]

**Comment by omarespejel** [comment]
@SimplyJuanjo thank you! 🤗 

@sgugger LGTM 👍

**Review by sgugger — APPROVED** [review]
Thanks a lot for your PR!

**Pull request merged**